src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -363,6 +367,11 @@ class AccessibleView extends Disposable {
 				e.stopPropagation();
 			}
 		}));
+		disposableStore.add(addDisposableListener(this._toolbar.getElement(), 'keydown', (e: KeyboardEvent) => {
+			if (e.key === 'Escape') {
```

Comment:
You could use known enums here instead of a string literal, [example](https://github.com/microsoft/vscode/blob/95ec90210b31683f6a0572c770780b0ee71d910e/src/vs/workbench/contrib/preferences/browser/keybindingsEditor.ts#L1000-L1002)

```suggestion
		disposableStore.add(addDisposableListener(this._toolbar.getElement(), DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
			const keyboardEvent = new StandardKeyboardEvent(e);
		        if (keyboardEvent.equals(KeyCode.Escape)) {
```